src/entities/ProductSchema.ts

```diff
@@ -0,0 +1,31 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+
+export const ProductSchema = z.object({
+  name: z
```

Comment:
Maybe (if you have enough time), you can extract these validations in a separate file so you will be able to reuse them (somewhere in the future :D)

Reply:
Done